src/components/footer.js

```diff
@@ -10,7 +10,7 @@ const Footer = () => {
     const absolutePath = location.pathname.split("/")
 
     return(
-        <footer className={`footer m6-top ${location.pathname === '/login' || location.pathname === '/signup' || absolutePath[1] === "playlist" ? "display-none" : ""} ${(location.pathname === "/liked-videos" && likes.length === 0)  || (location.pathname === "/watch-later" && watchlater.length === 0) || (absolutePath[1] === "playlists" && playlists.length < 4 ) ? "fixed bottom-0 full-width" : "" }`}>
+        <footer className={`footer m6-top ${location.pathname === '/login' || location.pathname === '/signup' || absolutePath[1] === "playlist" || location.pathname === "/history" ? "display-none" : ""} ${(location.pathname === "/liked-videos" && likes.length === 0)  || (location.pathname === "/watch-later" && watchlater.length === 0) || (absolutePath[1] === "playlists" && playlists.length < 4 ) ? "fixed bottom-0 full-width" : "" }`}>
```

Comment:
instead you can create custom hook which will return response to show this footer or not